.github/workflows/_build.yml

```diff
@@ -194,6 +188,10 @@ jobs:
           disable-animations: true
           script: |
             sudo pip3 install Pillow==9.0.1
+            adb logcat -c
```

Comment:
we didn't have access to the logs before, so this will give us some more insight into CI failures